api-tests/data/practitioner-service/linkperson.js

```diff
@@ -0,0 +1,40 @@
+const createFirstPerson = {
+    "prefix": "Mr",
+    "givenName": "Adam",
+    "familyName": "Rich"
+}
```

Comment:
Hi @ben0deinde  can you use the function, randomStringGenerator from utils file in order to generate random users for your tests please. Please refer to the other spec files create person, create site that are already using it, thank you